pkg/karmadactl/cmdinit/utils/examples.go

```diff
@@ -143,51 +153,41 @@ func GenExamples(path string) {
-| Step 3: Create karmada agent                                                                                                                                                 |
-|                                                                                                                                                                              |
-| (In member kubernetes)~#  MEMBER_CLUSTER_NAME="demo"                                                                                                                         |
-| (In member kubernetes)~#  sed -i "s/{member_cluster_name}/${MEMBER_CLUSTER_NAME}/g" karmada-agent.yaml                                                                       |
-| (In member kubernetes)~#  kubectl create -f karmada-agent.yaml                                                                                                               |
-|                                                                                                                                                                              |
-| Step 4: Show members of karmada                                                                                                                                              |
-|                                                                                                                                                                              |
-| (In karmada)~# kubectl  --kubeconfig /etc/karmada/karmada-apiserver.config get clusters                                                                                      |
-└──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
-`))
+	fmt.Printf(`Member kubernetes join karmada control plane.
+
+Push mode
+                                                                                                                                                                             
+Step 1: Use kubectl-karmada join to add members to the control panel. --cluster-kubeconfig is members kubeconfig.
+(In karmada)~# MEMBER_CLUSTER_NAME=%scat ~/.kube/config  | grep current-context | sed 's/: /\n/g'| sed '1d'%s
+(In karmada)~# kubectl-karmada  --kubeconfig %s/karmada-apiserver.config  join ${MEMBER_CLUSTER_NAME} --cluster-kubeconfig=$HOME/.kube/config
+
+Step 2: Create member kubernetes kubeconfig secret
+(In member kubernetes)~# kubectl create ns karmada-system
+(In member kubernetes)~# kubectl create secret generic ${MEMBER_CLUSTER_NAME}-kubeconfig --from-file=${MEMBER_CLUSTER_NAME}-kubeconfig=$HOME/.kube/config  -n karmada-system 
+
+Step 3: Create karmada scheduler estimator
+(In member kubernetes)~# sed -i "s/{{member_cluster_name}}/${MEMBER_CLUSTER_NAME}/g" %s/karmada-scheduler-estimator.yaml
+(In member kubernetes)~# kubectl create -f  %s/karmada-scheduler-estimator.yaml
```

Comment:
The `step 2` and `step 3` are about to deploy `karmada-scheduler-estimator`, right? I think this information shouldn't present here, because it is not the mandatory component.